### ISSUE TITLE
Getting element is covered error even when there is a matching element present that is not covered

### ISSUE THREAD

**Issue by sguptatw** [getgauge/taiko#652]
### Expected behavior
Getting element is covered error even when there is a matching element present that is not covered

### Actual behavior
✘ Error: Element matching text "Bengaluru" is covered by other element, run `.trace` for more info.

### Steps to reproduce
1. open the REPL
```
> openBrowser()
 ✔ Browser opened
> goto("bookmyshow.com")
 ✔ Navigated to URL http://bookmyshow.com
//Click deny on location popup
//Wait for the cities popup to appear
> click("Bengaluru")
 ✘ Error: Element matching text "Bengaluru" is covered by other element, run `.trace` for more info.
```
> There is an element "Bengaluru" that is not covered
<img width="1198" alt="Screenshot 2019-06-26 at 3 22 37 AM" src="https://user-images.githubusercontent.com/6310197/60136440-ca4c6f80-97c1-11e9-8ac7-df1b86e76bab.png">

### Versions
```
commit - 7a49c17886d51b619dcfa53717fd88518d6499dc
```

**Comment by Debashis9012** [comment]
This issue has been verified and found fixed.
Tested version: `271b55182385ee895e6b3a284d1a4aea1bdda3e0`

**Comment by supraja-kr** [comment]
I am facing this issue still, even on pulling the latest versions

**Comment by Debashis9012** [comment]
@supraja-kr  May we know which version of taiko you are using?

**Comment by acecyq** [comment]
Same here I am facing the same issues

**Comment by CzarScar** [comment]
I am also facing this issue. I am using the latest release 1.0.0

**Comment by luciferankon** [comment]
@CzarScar @supraja-kr with the 1.0.0 version of taiko the script below works perfectly for me.
```
const { openBrowser, goto, click, closeBrowser } = require('taiko');
(async () => {
    try {
        await openBrowser();
        await goto("bookmyshow.com");
        await click("Bengaluru");
    } catch (error) {
        console.error(error);
    } finally {
        await closeBrowser();
    }
})();
```
Could you please provide some html and script snippet to replicate this issue?

**Comment by advait012** [comment]
@Debashis9012 I am still facing the same issue and Taiko version is 1.0.2.Please help!!

**Comment by zabil** [comment]
The fix is not released yet it will only be available in 1.0.3

**Comment by sanjeeth-07** [comment]
This was resolved after adding `waitFor()` before subsequent clicks.
`await waitFor(1000) `
